tests/nnapi/specs/V1_1/transpose.mod.py

```diff
@@ -1,14 +1,16 @@
 model = Model()
 i1 = Input("input", "TENSOR_FLOAT32", "{1, 2, 2, 1}")
-perms = Parameter("perms", "TENSOR_INT32", "{4}", [0, 2, 1, 3])
+perms = Input("perms", "TENSOR_INT32", "{4}")
```

Comment:
Please do not modify this test file.

Reply:
Reverted the changes, with reasoning given in Issue #1875 . Thank you for the discussion.